core/src/main/resources/hudson/PluginManager/installed_pl.properties

```diff
@@ -20,7 +20,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 Installed\ Plugins=Zainstalowane wtyczki
```

Comment:
`core/src/main/resources/hudson/PluginManager/installed.jelly` no longer contains an "Installed Plugins" string with a capital I, so this line can be deleted:

```suggestion
```